app/components/Search/SearchPage.js

```diff
@@ -59,7 +59,7 @@ const SearchPage = (props: Props) => {
   const handleSelect = (result: SearchResult) => {
     setQuery('');
     setSelectedIndex(0);
-    props.handleSelect(result);
+    props.handleSelect(result.username ?? '');
```

Comment:
Do you know if `username` in `SearchResult` can ever be undefined? It seems to pass flow fine without being optional, and I can't really think of a situation where it would be undefined.

Reply:
Yeah, it happens when you search for other things, such as events, meetings. However, in this search it will filter for only users. The nullish check is only needed because of the SearchResult flow type.

Branch is up to date now btw

Reply:
Without the null check, flow fails for me at least ✌🏻

Reply:
Ohh, right. It's the same type for all search results. That makes sense.